projects/angular/src/accordion/stepper/stepper-panel.spec.ts

```diff
@@ -124,6 +132,18 @@ describe('ClrStep Reactive Forms', () => {
 
       expect(input.focus).toHaveBeenCalled();
     }));
+
+    it('should have the aria-level attribute set to 4', () => {
```

Comment:
```suggestion
    it('should allow the aria-level attribute to be set to an explicit value', () => {
```

This test is for an explicit heading value, so I think the test description should say that.